install-dependencies.sh

```
@@ -224,6 +224,216 @@ function ensure_cargo() {
+  
+  ensure_nodejs
+  npm install -g sindri@latest
+}
+
+# taplo (TOML formatter)
+function ensure_taplo() {
+  check_cmd taplo && return
+
+  case $OS in
+  Linux)
+    local cmd=$(
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)